nscplugin/src/main/scala/scala/scalanative/nscplugin/NirGenExpr.scala

```diff
@@ -1304,11 +1225,28 @@ trait NirGenExpr { self: NirGenPhase =>
       buf.elem(Type.Byte, ptr, Seq(offset), unwind)
     }
 
+    def genRawWordOp(app: Apply, code: Int): Val = (app, code) match {
+      case (Apply(_, Seq(leftp, rightp)), ADD_RAW_WORDS) =>
+        buf.bin(Bin.Iadd, Type.Word, genExpr(leftp), genExpr(rightp), unwind)
+      case (Apply(_, Seq(leftp, rightp)), SUB_RAW_WORDS) =>
+        buf.bin(Bin.Isub, Type.Word, genExpr(leftp), genExpr(rightp), unwind)
+      case (Apply(_, Seq(leftp, rightp)), MULT_RAW_WORDS) =>
+        buf.bin(Bin.Imul, Type.Word, genExpr(leftp), genExpr(rightp), unwind)
+      case (Apply(_, Seq(leftp, rightp)), DIV_RAW_WORDS) =>
+        buf.bin(Bin.Sdiv, Type.Word, genExpr(leftp), genExpr(rightp), unwind)
+      case (Apply(_, Seq(leftp, rightp)), DIV_RAW_WORDS_UNSIGNED) =>
+        buf.bin(Bin.Udiv, Type.Word, genExpr(leftp), genExpr(rightp), unwind)
+      case _ =>
+        abort(
+          s"Unknown word operation #$code : " + app +
+            " at: " + app.pos)
+    }
```

Comment:
There is lots of duplication in the code above. You can extract the arguments from app and then match on `code` only. i.e.:
```
def genRawWordOp(app: Apply, code: Int): Val = {
  val Apply(_, Seq(leftp, rightp)) = app
  val bin = code match {
     case ADD_RAW_WORDS => Bin.Iadd
     ...
     case _ => abort("...")
  }
  buf.bin(bin, Type.Word, genExpr(leftp), genExpr(rightp), unwind)
}
```

Reply:
This file is the only one that scalafmt gives up on, please vertically align cases by `=>`.